deployments/bridges/rialto-millau/entrypoints/relay-messages-to-millau-generator-entrypoint.sh

```diff
@@ -46,6 +46,7 @@ do
 		$SEND_MESSAGE \
 			--lane $SECONDARY_MESSAGE_LANE \
 			--origin Target \
+			--dispatch-fee-payment at-target-chain \
```

Comment:
Would that always work? Do we have enough funds in the companion account?

Reply:
The dispatch origin on target chain is //Dave, so it should have enough funds. I can't guarantee if it'll be enough for all daily messages, but probably it will.

Actually, I just had a thought - this PR should have no effect on our chains, because `multiplier` [here](https://github.com/paritytech/substrate/blob/19c6dd2f2ab8dea8d5734f2a0f246c935b76e75e/frame/transaction-payment/src/lib.rs#L473) is always zero. But actually it will have an effect, because multiplier is not used [here](https://github.com/paritytech/parity-bridges-common/blob/20d08204a2a93fef5c93c70c488f9049086fdb97/bin/runtime-common/src/messages.rs#L531). So the answer to your question would be: (1) yes we have enough funds because amount required for this PR is `0` (2) current implementation has an issue, though, so it'll be non-zero - I'll fix it in a parallel PR.

Reply:
The alternative would be to change multiplier in Millau/Rialto to non-zero. E.g. to `1`, or even use `1` iniitally and `SlowAdjustingFeeUpdate` from polkadot runtime. I've been using hardcoded version for some experiments some time ago, but probably changing to somewhat similar to production code will be better. Anyway - I'm going do that in a separate PR - it's not a trivial change.